neo-cli/CLI/MainService.Logger.cs

```diff
@@ -71,43 +93,89 @@ private void OnLog(string source, LogLevel level, object message)
+                        case LogLevel.Fatal: logColor = FatalColor; logLevel = "FATAL"; break;
+                        case LogLevel.Info: logColor = KeyColor; logLevel = "INFO"; break;
+                        case LogLevel.Warning: logColor = WarningColor; logLevel = "WARN"; break;
+                        default: logColor = InfoColor; logLevel = "INFO"; break;
                     }
+                    logColor.Apply();
+                    Console.Write(logLevel + " ");
+                    InfoColor.Apply();
 
-                    Console.WriteLine(log);
+                    Console.Write($"{log} {messages[0]}");
+                    for (var i = 0; i < 35 - messages[0].Length - logLevel.Length; i++) Console.Write(' ');
+                    for (var i = 1; i < messages.Length; i++)
+                    {
+                        if (messages[i].Length > 20)
+                        {
+                            messages[i] = $"{messages[i][..10]}...{messages[i][(messages[i].Length - 10)..]}";
+                        }
+                        if (i % 2 == 0)
+                        {
+                            InfoColor.Apply();
+                            Console.Write($"={messages[i]} ");
+                        }
+                        else
+                        {
+                            logColor.Apply();
+                            Console.Write($" {messages[i]}");
+                        }
+                    }
                     currentColor.Apply();
+                    Console.WriteLine();
                 }
 
-                if (!string.IsNullOrEmpty(Settings.Default.Logger.Path))
+                if (string.IsNullOrEmpty(Settings.Default.Logger.Path)) return;
+                var sb = new StringBuilder(source);
+                foreach (var c in GetInvalidFileNameChars())
+                    sb.Replace(c, '-');
+                var path = Combine(Settings.Default.Logger.Path, sb.ToString());
+                Directory.CreateDirectory(path);
+                path = Combine(path, $"{now:yyyy-MM-dd}.log");
+                try
                 {
-                    StringBuilder sb = new StringBuilder(source);
-                    foreach (char c in GetInvalidFileNameChars())
-                        sb.Replace(c, '-');
-                    var path = Combine(Settings.Default.Logger.Path, sb.ToString());
-                    Directory.CreateDirectory(path);
-                    path = Combine(path, $"{now:yyyy-MM-dd}.log");
-                    try
-                    {
-                        File.AppendAllLines(path, new[] { $"[{level}]{log}" });
-                    }
-                    catch (IOException)
-                    {
-                        Console.WriteLine("Error writing the log file: " + path);
-                    }
+                    File.AppendAllLines(path, new[] { $"[{level}]{log}" });
```

Comment:
```suggestion
                    File.AppendAllLines(path, new[] { $"[{level}]{log} {message}"});
```
`message` is missing here, how could everything work well on your machine:joy:

Reply:
My bad, i must have checked the wrong log file